src/.nuget/Microsoft.NETCore.ILAsm/runtime.Android.Microsoft.NETCore.ILAsm.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@gkhanna79: So the build is successful, including packages, once `packageTargetOsGroup` for Android is removed from `dir.props` and conditionals added to `Microsoft.NETCore.Runtime.CoreCLR.pkgproj`

So my questions are:

1. What elements distinguish one package group from another? Or: What factors determine the creation of a new OS group?
2. Is it possible that a separate package group would be needed for Android in the future?